client/client.go

```diff
@@ -766,7 +766,8 @@ func (r *NotaryRepository) Update(forWrite bool) (*tufclient.Client, error) {
 		}
 		return nil, err
 	}
-	return c, nil
+	r.tufRepo = repo
+	return nil, nil
```

Comment:
Should we change the function signature to only return `error`, since we never return a pointer to a tuf client now?